CHANGELOG.md

```diff
@@ -1,12 +1,12 @@
 Unreleased:
 
 - Updated JDK to Corretto 1.8.342
-- Upgraded Jackson Databind Component to 2.13.4.2
+- Upgraded Jackson Components to 2.15.2
 
 
 *1.3.3-GRESHAM-28:
 - Updated JDK to Corretto 1.8.332
-- Uprgaded Jackson Components to 2.13.3
+- Upgraded Jackson Components to 2.15.2
```

Comment:
-28 has already been released, has it not? How can we historically upgrade on a released version?